CONTRIBUTING.md

```diff
@@ -26,14 +26,16 @@ offer. Note that the first time you run `make` various dependent libraries and t
 downloaded and installed which can be somewhat time consuming. Be patient.
 
 Note that if you edit a `.proto` or `.ts` file, you will need to manually regenerate the associated `.pb.{go,cc,h}` or `.js` files using `go generate ./...`.
-`go generate` requires the typescript transpiler `tsc`, which you can get from [typescript](http://www.typescriptlang.org/) or using `npm` with:
-`npm install -g typescript`
-`go generate` also requires a typescript linter `tslint`, which you can get from [tslint](https://github.com/palantir/tslint) or using `npm` with:
-`npm install -g tslint`
-If you don't have npm, it comes with node. To get it via homebrew:
+`go generate` requires a collection of node modules which are installed via npm. If you don't have npm, it comes with node. To get it via homebrew:
```

Comment:
We install them separately in `build/builder.sh` and `devbase/Dockerfile`; does this mean we should remove those and only install nodejs?

On macports I can confirm that installing `nodejs` by itself does not install `npm`.